css/base/variables.css

```diff
@@ -137,6 +137,7 @@ body {
   --border: var(--border-width) var(--border-style) var(--border-color);
   --border-large: var(--border-width-large) var(--border-style) var(--border-color);
   --border-dark: var(--border-width) var(--border-style) var(--border-color-dark);
+  --border-transparent: 1px solid transparent;
```

Comment:
This should use our default border variables `--border-transparent: var(--border-width) var(--border-style) transparent;`

This is so that if other parts of the item where this is used depend on the border being a certain thickness, we won't be throwing that off by specifying this border as 1px. E.G. if the border is no longer transparent on :hover.

Reply:
Unfortunately adding `border-width: none` would undo what has been added.
So end up back at square one: no visible border on search input with WHCM.

The transparent border is invisible, as it is there from the start it should not affect layout, or not noticeably so.

but some quick responsive testing should hopefully pick out any anomalies.

Reply:
Thanks very much @markconroy and @waako the branch has been updated accordingly.